addons/gamepad/style.css

```diff
@@ -40,6 +40,7 @@
   will-change: transform;
   image-rendering: optimizeSpeed;
   image-rendering: crisp-edges;
+  image-rendering: -moz-crisp-edges;
```

Comment:
According to MDN, `crisp-edges` is enough on Firefox 65+

Reply:
Also, I think garbo meant `text-rendering: optimizeSpeed`? I don't think `optimizeSpeed` is valid here.